BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoiceViewController.swift

```diff
@@ -8,36 +8,22 @@
 import UIKit
 import GiniBankAPILibrary
 import GiniCaptureSDK
-/**
- Delegate protocol for `DigitalInvoiceViewController`.
- */
-public protocol DigitalInvoiceViewControllerDelegate: AnyObject {
-    
```

Comment:
[Clarififcation] @vizaknaidavid, why did we remove it?

Reply:
It was not used. These methods were replaced by `DigitalInvoiceCoordinatorDelegate`